### ISSUE TITLE
Naming of struct fields wrong

### ISSUE THREAD

**Issue by rfalke** [yegord/snowman#225]
Previous version: 393ccfdd4b99
Current version: 6fed71ccb650

I get a lot of such diffs when I compare these versions:
```
--- a/from_boomerang/asgngoto/ia32_elf_by_boomerang/by_snowman.c
+++ b/from_boomerang/asgngoto/ia32_elf_by_boomerang/by_snowman.c
@@ -82,7 +82,7 @@ void fun_80485a4(int32_t a1, int32_t a2, int32_t a3) {
 
 struct s0 {
     signed char[4] pad4;
-    int32_t f4;
+    int32_t 0x4;
 };
 
 void fun_8048651(int32_t a1) {
@@ -93,7 +93,7 @@ void fun_8048651(int32_t a1) {
     if (eax2) {
         eax2();
     }
-    goto ebp3->f4;
+    goto ebp3->0x4;
 }
 
 int32_t s_wsle = 0x804861a;
```

It looks like something is wrong here.

**Comment by yegord** [comment]
The `f` version of the output is the intended one.
There was a regression that led to invalid field names, fixed in acac978f7c6970584077dd6ab8b3747a35b721a2.

**Comment by rfalke** [comment]
My bad. Looks like the build got somehow messed up. A clean rebuild doesn't show this behavior.
